lib/java/desktop-environment/src/main/java/org/enso/desktopenvironment/directories/DirectoriesException.java

```diff
@@ -0,0 +1,15 @@
+package org.enso.desktopenvironment.directories;
```

Comment:
I'd flatten the directories and put all API into `org.enso.desktopenvironment` package.